game/Submods/NSFW Submod/nsfw_stories.rpy

```diff
@@ -272,63 +272,62 @@ label nsfw_erotic_story_thepen:
-    m 2eub "So...I was in my room, searching for something that will make masturbating more exciting."
-    m 3eua "When I found a pen sitting on the desk."
+    m 2eub "So...I was in my room, searching for something that would make it more exciting..."
+    extend 3eua "when I found my pen sitting on the desk."
     if inspired_monika == True:
-        m 3rkblb "My mind immediately went back to when I overheard that Yuri had used the main character's pen when she masturbated."
-        m 3rtblu "I wondered if it was that good of a feeling, or if she was just so obsesseed with the main character that anything would have done it."
+        m 3rkblb "My mind immediately went back to when I overheard that Yuri had used the main character's pen to stimulate herself."
+        m 3rtblu "I wondered if it was that good of a feeling, or if she was just so obsessed with him that anything would have done it."
     else:
         m 3rkblb "My mind immediately filled with thoughts of using it."
     m 1hublb "Ahaha! I can't see your face, but I can already tell you're asking why, right?"
     m 1eubla "Well, I understand you might think it's strange."
     m 3tubla "But when you don't have anything but your fingers to masturbate with..."
-    m 3hublb "You get kind of desperate. Ahaha~"
+    extend 3hublb "you get kind of desperate. Ahaha~"
     m 4eublb "Anyway, I took the pen and walked back to the bed."
-    m 4tublb "I first stripped down to my underwear, and then laid down on the bed."
+    m 4tublb "I first stripped down to my underwear, and then lay down."
     m 4tublu "I spread my legs {i}just a little{/i}, and started using the pen on my...sensitive parts."
-    m 2tublb "On it's own, it wasn't that special."
-    m 5tublb "But then I started to fantaize about you..."
-    m 5dubla "I closed my eyes."
-    m 5dubsb "I pictured your face sitting directly above mine, with your index finger rubbing between my legs."
-    m 5dkbsa "In my mind you leaned down to kiss me, and I could almost feel your lips pressed against mine."
-    m 2tkbsb "I felt my body getting hot, and the area down there was feeling more moist than before..."
+    m 2tublb "On its own, it wasn't that special."
+    m 5tublb "But then I started to fantasize about you..."
+    m 5dubla "I closed my eyes, "
+    extend 5dubsb "and I pictured your face sitting directly above mine, with your index finger rubbing between my legs."
+    m 5dkbsa "In my mind, you leaned down to kiss me, and I could almost feel your lips pressed against mine."
+    m 2tkbsb "I felt my body getting hot, and the area down there began to feel a lot more moist than before..."
     m 2gkbsa "I started rubbing myself faster, and faster~"
-    m 2tkbsu "I felt you getting rougher with me as your finger was making it's way up and down my pussy."
+    m 2tkbsu "I imagined that you getting rougher with me as your finger was making its way up and down my pussy."
```

Comment:
"I imagined that you **were** getting rougher with me..."

Reply:
Oops, good catch